src/winetricks

```diff
@@ -9439,6 +9439,34 @@ verify_dotnet48()
 
 #----------------------------------------------------------------
 
+w_metadata dotnetcore2 dlls \
+    title="MS .NET Core Runtime 2.1 LTS" \
+    publisher="Microsoft" \
+    year="2020" \
+    media="download" \
+    conflicts="" \
+    file1="dotnet-runtime-2.1.17-win-x86.exe" \
+    installed_file1="$W_PROGRAMS_WIN/dotnet/dotnet.exe"
+
+load_dotnetcore2()
+{
+    # Official version, see https://dotnet.microsoft.com/download/dotnet-core/2.1
+    w_download https://download.visualstudio.microsoft.com/download/pr/73a0ea97-57d6-4263-ac36-ea3a9b373bcf/cd5f7e08e749c1d3468cdae89e4518de/dotnet-runtime-2.1.17-win-x86.exe 706a7f0ad998c3c1b321e89e4bcd6304bef31c95c83eda119e8d4adccccbc915
+
+    w_try_cd "$W_CACHE"/"$W_PACKAGE"
+    w_try "$WINE" dotnet-runtime-2.1.17-win-x86.exe
+
+    case "$W_ARCH" in
```

Comment:
Not a blocker (and I know you copied this from others) but I'd prefer `if [ "$W_ARCH" = "win64"; then ...`.